main.go

```diff
@@ -116,10 +125,10 @@ func (c *graphiteCollector) processLines() {
 
 func (c *graphiteCollector) processLine(line string) {
 	line = strings.TrimSpace(line)
-	log.Debugf("Incoming line : %s", line)
+	level.Debug(c.logger).Log("msg", "Incoming line", "line", line)
 	parts := strings.Split(line, " ")
 	if len(parts) != 3 {
-		log.Infof("Invalid part count of %d in line: %s", len(parts), line)
+		level.Info(c.logger).Log("msg", "Invalid part count", "length", len(parts), "parts", line)
```

Comment:
I would prefer this to be

```suggestion
		level.Info(c.logger).Log("msg", "Invalid part count", "parts", len(parts), "line", line)
```

since we use `line` as the key when logging the whole received line everywhere else